app/modules/encounters/models.py

```diff
@@ -166,7 +166,7 @@ def get_time(self, sighting_fallback=True):
         if self.time:
             return self.time
         if self.sighting and sighting_fallback:
-            return self.sighting.get_time()
+            return self.sighting.time
```

Comment:
not sure why this change was needed. I avoid accessing internal variables within objects outside of the class as it makes later refactoring much more difficult. There usually has to be a reason not to use a method and this seems to be going the opposite way.